Makefile

```diff
@@ -74,14 +74,14 @@ $(BINARIES): bin/%:
 mod-tidy:
 	$(Q)for mod in $$(find . -name go.mod); do \
 	    echo "Tidying $$mod..."; ( \
-	        cd $$(dirname $$mod) && go mod tidy \
+	        cd $$(dirname $$mod) && go mod tidy -go=$$(grep -E "^go\s+[1-9]\.[0-9]+$$" go.mod | sed 's/^go //') \
             ) || exit 1; \
 	done
 
 mod-verify:
 	$(Q)for mod in $$(find . -name go.mod); do \
 	    echo "Verifying $$mod..."; ( \
-	        cd $$(dirname $$mod) && go mod verify | sed 's/^/  /g' \
+	        cd $$(dirname $$mod) && go mod verify -go=$$(grep -E "^go\s+[1-9]\.[0-9]+$$" go.mod | sed 's/^go //') | sed 's/^/  /g' \
```

Comment:
`go mod verify` doesn't have `-go` command line option according to [its help page](https://go.dev/ref/mod#go-mod-verify).

Reply:
You're right! Updated.